project_2.html

```diff
@@ -0,0 +1,603 @@
+			Remember that the frequency of the waveform is the same as the pitch of the buzzer sound.
+			<br><br>
+		    </p>
+
+			<h4 class="mb-3 fw-light" id="pj-heading-NE555PIC" style="display: inline;">Pinout Diagram</h4>	
+			<p class="mb-3">
+				We will use the <strong>NE555P</strong>, a variant of the 555 Timer. Its pinout diagram is shown below. 
+				<img class="img-fluid my-3 me-5 float-start" style="max-width: 400px;"
+					src="assets/images/visuals/ops_pinout_ne555p.png">
+			</p>
+
+			<p class="mb-5">
+					The <strong>pinout diagram</strong> identifies each contact or pin of the IC. Usually, a datasheet will 
+					include a pinout and a table that matches each pin with a description. The pinout may 
+					change depending on the IC’s packaging, which the datasheet will indicate. For this 
+					project, the IC package is DIP.
+					<br>
+					Note that the organization of the pinout differs from the schematics below. The true location of the ICs pins is shown by the 
+					pinout, not the schematic. Use the notch on a DIP-style IC to find its correct orientation.	
+					<br><br>	
+			</p>
+
+			<h4 class="mb-3 fw-light" id="pj-heading-NE555PIC" style="display: inline;">How It Works</h4>	
+			<p class="mb-5">
+				Learn more about 555 Timer’s functionality in the lecture material <span class="text-blue">here [Webpage Link]</span>.
+				<br><br>
+			</p>
+
+			<h3 class="mb-3" id="pj-heading-pcb">Printed Circuit Board (PCB)</h3>	
+			<p class="mb-3">
+				<img class="img-fluid my-3 me-5 float-end m-lg-3"  style="max-width: 400px;" 
+					src="assets/images/visuals/ops_proj2_pcb.png">
+				Breadboards and protoboards are great for prototyping simple circuits. However, when electronics are more complex or are ready for production, we typically use <strong>printed circuit boards </strong>
+				or <strong>PCBs</strong>. Nearly every device, from a digital alarm clock to a laptop computer, uses a PCB. It is a board designed 
+				with CAD software, that connects individual components. 
+				<br><br>
+				The benefits of PCBs over prototyping boards are numerous: They’re more compact, more reliable, and enable 
+				complex circuit design. Above all, they are easy to mass produce.	
+				
+			</p>
+
+		
+
+			<p class="mb-3">
+				For this project, you will solder the 555 Piano to a PCB we designed in a software called 
+				KiCAD (as seen below). If you are not enrolled in the course, you may alternatively use a perfboard.
+				<br>
+				<img class="img-fluid my-3 me-5 float-start" style="max-width: 700px;"
+					src="assets/images/visuals/ops_kicad_example.png"> 
+				</p>
+
+			<p class="mb-5">
+				Soldering to a PCB is much like soldering to a protoboard, and you may even find it easier. 
+				For more soldering tips visit our workshop material <a class="link-primary" 
+				href="https://docs.google.com/presentation/d/11AWxHFSlhvF4bWnvbFpBkxV5KOSYh7OgkUuCPrOEb2k/edit?usp=sharing" target="_blank" rel="noopener noreferrer">here</a>.

```

Comment:
![image](https://github.com/uci-ieee/ops-webpages-2023-2024/assets/45702229/1c4cf5f8-cbfd-4ea8-b4fb-11c6f054c396)

Now that there's a workshops page, please update the hyperlink to redirect to workshops.html